routes/recipes/recipes.js

```diff
@@ -0,0 +1,82 @@
+const recipes = require('express').Router()
+
+//let query = req.query
+let recipeArr = [
+    {name: "PBJ",
+    ingredients: ['bread','peanut butter', 'jelly'
+],
+    directions: 'slap all that together with the peanut butter and jelly between 2 slices of bread'},
+
+    {name: 'cereal n milk',
+     ingredients: ['cereal', 'milk'],
+     directions: 'pour ceral into a bowl FIRST, then pour milk'},
+    ]
+    // if(recipeArr[i]['ingredients'].includes(req.params.id)){
+
+    // }
+// let recipeObj = [{
+//     name: "",
+//     ingredients: [],
+//     directions: ""
+// }]
+
+recipes.get('/', (req,res) => {
+
+    res.json(recipeArr)
+})
+recipes.post('/',(req,res) => {
+    recipeArr.push(req.body)
+    res.json({
+        status:200,
+        message: "posted new recipe",
+        newRecipe: req.body
+    })
+
+})
+recipes.delete('/:id',(req,res) => {
```

Comment:
You are not actually deleting the recipe here - you need to:
1. Find the user that matches the params 
2. write some kind of function (think splice) to remove that user from the array of all users.